tools/azsdk-cli/Azure.Sdk.Tools.Cli/Helpers/AzureOpenAIClientHelper.cs

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Azure.Core;
+using OpenAI;
+using System.ClientModel;
+using System.ClientModel.Primitives;
+
+namespace Azure.Sdk.Tools.Cli.Helpers;
+
+/// <summary>
+/// Helper class to create OpenAI clients configured for Azure endpoints with authentication
+/// </summary>
+public static class AzureOpenAIClientHelper
+{
+    private const string PlaceholderApiKey = "not-used";
```

Comment:
Why create a field for this? I worry that scanners might flag this thinking it is a secret. I would suggest just hard-coding this value inline.